modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -162,21 +182,20 @@ Mule persists a list of all records that succeed or fail to process through each
 
 At the end of this phase, the batch job instance completes and, therefore, ceases to exist.
 
-==== Refine Batch Steps Processing
+=== On Complete
 
-Beyond simple processing of records, there are several things you can do with records within a batch step.
+During this phase, you can configure the runtime to create a report or summary of the records it processed in a given batch job instance. This phase provides system administrators and developers insight into which records failed or succeeded. This phase does not process or provide access to individual records, nor does it pass processed records to downstream processors in the flow.
 
-* You can apply filters by adding acceptExpressions within each batch step to prevent the step from processing certain records.
-For example, you can set a filter to prevent a step from processing any records which failed to process in the preceding step.
+Because Mule processes a batch job as an asynchronous, one-way flow, the results of batch processing do not return to the flow, nor do the results return as a response to a caller. Any event source that feeds data into a batch job instance must be one-way, not request-response.
 
-* You can use a batch aggregator processor to aggregate records in groups, sending them as bulk upserts to external sources or services.
-For example, rather than upserting each contact (that is, a record) in a batch to Google Contacts, you can configure a batch aggregator to accumulate, say, 100 records, then upsert all of them to Google Contacts in one chunk.
 
-See xref:batch-filters-and-batch-aggregator.adoc[Refining Batch Steps Processing] for more information.
+As a best practice, configure a mechanism for reporting on failed or successful records to facilitate further action where required. During the On Complete phase, you can perform either of these tasks:
 
-=== On Complete
-
-During this phase, you can optionally configure the runtime to create a report or summary of the records it processed for the particular batch job instance. This phase exists to give system administrators and developers some insight into which records failed to address any issues that might exist with the input data.
+* Reference the result object for the batch job instance from elsewhere in the Mule application to capture and use batch metadata, such as the number of records that failed to process in a particular batch job instance.
++
+TODO: HOW DO WE DO THIS?
```

Comment:
TODO: HOW DO WE 

Reference the result object for the batch job instance from elsewhere in the Mule application to capture and use batch metadata, such as the number of records that failed to process in a particular batch job instance.